src/script.ts

```diff
@@ -66,7 +66,7 @@ export interface ScriptConfig extends ScriptReference {
    * - "if-file-deleted": If an input file has been deleted, and before restoring from
    *   cache.
    */
-  clean: AstNode<boolean | 'if-file-deleted'> | undefined;
+  clean: boolean | 'if-file-deleted';
```

Comment:
Wondering if it would make sense to be an AstNode but excluding the undefined?

Reply:
So far I've avoided mutating or creating fake AstNodes. What would the range be? Would we make a full fake property with a fake key and value?

A problem we can chase if anything ever wants to refer to clean's location